config/data.json

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
We need to get proper linting set up in this project. Tabs are unacceptable in source files.